fix: revive nonIsolatedModules (#172)
toomuchdesign/next-page-tester: README.md

| **nextRoot**                   | Absolute path to Next.js root folder                                               | `string`                          | _auto detected_ |
| **dotenvFile**                 | Relative path to a `.env` file holding [environment variables][next-docs-env-vars] | `string`                          | -               |
| **wrapper**                    | Map of render functions. Useful to decorate component tree with mocked providers.  | `{ Page?: NextPage => NextPage }` | -               |

> | **nonIsolatedModules**         | List of modules that should preserve identity between client and server context.   | `string[]`                        | []              |

Reviewer: What if we removed the negation? Something like `preserveModuleIdentity`?

Same could apply for our internal methods like: `unIsolatePredefinedJestModules ` , `unIsolateJestModules `..

Author: Hmm. Yeah naming this is confusing.... What about something less abstract for users to understand: `sharedModules` -- since their identity is "shared" between `client` and `server`.

Reviewer: Ok. Let's go with `sharedModules`!

Author: Done 👍

## FAQ

### How do I mock API calls in my data fetching methods?

> Because `next-page-tester` isolates modules between "client" and "server" context mocks that are created in test (client context) wont execute in data fetching methods (server context).

Reviewer: ```suggestion
Since `next-page-tester` isolates modules between "client" and "server" context, mocks created in test (client context) won't execute in data fetching methods (server context).
```
